regression/jbmc-strings/java_insert_char_array/test.desc

```diff
@@ -3,6 +3,6 @@ test_insert_char_array.class
 --refine-strings --string-max-length 1000
 ^EXIT=10$
 ^SIGNAL=0$
-assertion.* line 28.* SUCCESS$
-assertion.* line 30.* FAILURE$
+assertion.* line 26.* SUCCESS$
```

Comment:
Consider dropping specifying the line of the assertion (instead use its index) so this test is less volatile.